lib/ffmpeg/movie.rb

```diff
@@ -264,6 +269,51 @@ def any_streams_contain_audio?
       @any_streams_contain_audio ||= calc_any_streams_contain_audio
     end
 
+    # Get the max width and max height of all frames in the input movies and check if the resolutions of frames are consistent
+    def check_frame_resolutions
+      max_width = width || 0
+      max_height = height || 0
+      differing_frame_resolutions = false
+      last_dimensions = nil
+
+      unescaped_paths.each do |path|
+        local_movie = Movie.new(path)
+
+        # set max width and height from the movie metadata first
+        max_width = [max_width, local_movie.width].max
+        max_height = [max_height, local_movie.height].max
```

Comment:
Why set this based on the metadata if we're going to set it based on the actual output from the command below?

Reply:
This was retained from `calculate_interim_max_dimensions` incase theres a chance the metadata reports a higher resolution than the frames. No preference myself if its kept / removed!

Reply:
I don't see major harm and it's an incredibly "cheap" transaction, so should be fine